apps/angular/crud/src/app/components/todos/todos.component.html

```diff
@@ -0,0 +1,14 @@
+@if (todoStore.query.isFetched()) {
```

Comment:
I think todoStore.isPending() is better here. See https://tanstack.com/query/latest/docs/framework/angular/guides/queries#query-basics